### DISCUSSION TITLE
Is the conventional changelog action necessary?

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/defaang#57] [Tech]
We added a conventional changelog action at #25 and #6, but I'm not quite convinced that it is necessary.

It seems to clutter the notifications & emails a bit, and I'm not sure how much value it's adding to the project.

Thoughts?

**Comment by iShibi** [comment]
It should be removed since `defaang` is not a library so there is no mandatory requirement to keep a `CHANGELOG`. Also, you can easily auto generate release notes from the `Releases` tab. See: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository
